src/tainting/Taint.ml

```diff
@@ -374,6 +375,13 @@ end)
 
 type signature = Findings.t
 
+let show_signature s =
```

Comment:
I should probably use the `debug_` prefix, but until now I was using `_show_` to mean the same.